docs/tutorials/dqn.rst

```diff
@@ -83,29 +87,32 @@ Tianshou supports any user-defined PyTorch networks and optimizers but with the
     net = Net(state_shape, action_shape)
     optim = torch.optim.Adam(net.parameters(), lr=1e-3)
 
-You can also have a try with those pre-defined networks in :mod:`~tianshou.utils.net.common`, :mod:`~tianshou.utils.net.discrete`, and :mod:`~tianshou.utils.net.continuous`. The rules of self-defined networks are:
+You can have a try with those pre-defined networks in :mod:`~tianshou.utils.net.common`, :mod:`~tianshou.utils.net.discrete`, and :mod:`~tianshou.utils.net.continuous`. The rules of self-defined networks are:
```

Comment:
"It is also possible to use pre-defined networks in ..."

Besides, I would rather present first the pre-defined networks available, rather than the custom one. I think it is more appropriate to come later.

Reply:
I don't agree since I want to give users the maximum freedom to define the network.

Reply:
Sure, but I think it is better to start with predefined network, to provide a extremely quick and concise way to run a learning algorithm to get the feeling of the dataflow of Tianshou, how easy it is to train a model, and how fast is Tianshou. Then you introduce a more advanced example now that the user is convince the lib fits its need. But that's just my opinion.